source/io/module_javaScriptInvoke.js

```diff
@@ -309,13 +302,6 @@
             errorStatusPointer,
             errorCodePointer,
             errorSourcePointer) {
-            if (!isValidJavaScriptReturnType(returnUserValue)) {
```

Comment:
Why is this being removed?

Reply:
Because with the JSLI Object reference every possible JS value is a valid return value.

The problem is that this check is too broad, it needs to be specific to the return value the node is configured for not all possible return values.

For example if the JavaScriptInvokeNode return type is of type Double then we might only accept JavaScript Numbers as a supported type. We might also in the future support strings and automatically parse it as a double if that is a useful feature.

For the JSObjectReference return type any possible JavaScript value is valid.

Luckily we already have a check that verifies the return type to the JS type: https://github.com/ni/VireoSDK/pull/444/files/80cc0fabd7d9ebd3498ddecd9608fa9def77a580#diff-a7468c287af215525c8ba9f958336bd4R317

So the other check was redundant anyway and can be removed.